installation_and_upgrade/IBEX_upgrade.py

```diff
@@ -85,16 +86,22 @@ def _get_latest_existing_dir_path(release_dir, component):
         current_client_version = _get_latest_release_path(args.release_dir).split("\\")[-1]
         if args.release_suffix != "":
             current_release_dir += f"-{args.release_suffix}"
-        DIRECTORIES["EPICS"] = os.path.join(current_release_dir, "EPICS")
+
+        server_32bit_suffix = "32" if args.server_arch == "x86" else ""
```

Comment:
I think just `server_suffix` might be a better name as the suffix is not only applied in a 32 bit situation

Reply:
This might be for another ticket, but this would currently assume a release structure of `releases/11.0.0/EPICS` and `releases/11.0.0/EPICS32`  as opposed to e.g.  `releases/11.0.0/EPICS/win32-x86` and `releases/11.0.0/EPICS/windows-x64` (as in EPICS/${EPICS_HOST_ARCH}). For only two options this is probably not too important, but if we wanted to have static and debug deploys too then the other structure may be better